.env-example

```diff
@@ -0,0 +1,17 @@
+MAS_PUBLIC_WEBSITE_URL=http://<frontendhosturl>/:locale/
+MAS_CMS_URL=http://localhost:3000
+MAS_CMS_API_TOKEN=mytoken
+MAS_CONTENT_SERVICE_URL=http://localhost:8080/content-service/
+GOOGLE_API_KEY=xxxxx-xxxx
+GOOGLE_API_CX_EN=xxxx:xxxx
+GOOGLE_API_CX_CY=xxxx:xxxx
+STATSD_HOST=localhost
+STATSD_PORT=8125
+WEBCHAT_DOMAIN=xxxx.xxxx
+
+CAR_COST_TOOL_DOMAIN=localhost
+CAR_COST_TOOL_CAP_SUBSCRIBER_ID=''
+CAR_COST_TOOL_CAP_PASSWORD=''
+CAR_COST_TOOL_MAX_CARS_TO_COMPARE=3
+AUTH_USERNAME=admin
+AUTH_PASSWORD=password
```

Comment:
I am not sure if it is related to this PR, but we are not using MAS_CONTENT_SERVICE_URL (Hippo CMS) anymore.

Proof: 
https://github.com/search?q=org%3Amoneyadviceservice+MAS_CONTENT_SERVICE_URL&type=Code

Interestly we still have on the code the MAS_PUBLIC_WEBSITE_URL (old site), but there is only a public website assignment of the variable but not being used:

https://github.com/search?utf8=%E2%9C%93&q=org%3Amoneyadviceservice+MAS_PUBLIC_WEBSITE_URL&type=Code&ref=searchresults
https://github.com/moneyadviceservice/frontend/blob/473919b720d5d0e8b4286b595b3b746fc4a06ddd/config/initializers/core.rb#L9
And the public_website_connection variable is not being used in anywhere on the app besides the setup: https://github.com/search?utf8=%E2%9C%93&q=org%3Amoneyadviceservice+public_website_connection&type=Code&ref=searchresults

Wdyt that we should be actioning for this env vars?